src/components/subsidy-request-configuration/SubsidyRequestConfigurationContextProvider.jsx

```diff
@@ -23,13 +23,45 @@ const SubsidyRequestConfigurationContextProvider = ({ enterpriseUUID, children }
   );
 };
 
+const SubsidyRequestConfigurationContextProviderWrapper = ({ enableBrowseAndRequest, ...props }) => {
+  if (enableBrowseAndRequest) {
+    return <SubsidyRequestConfigurationContextProvider {...props} />;
+  }
+
+  const context = {
```

Comment:
`useMemo` with an empty dependency array? Only because `context` will be a new object reference on any subsequent re-renders of `SubsidyRequestConfigurationContextProviderWrapper`, and this is largely wrapping the whole app.